packages/composites/synth-index/README.md

```diff
@@ -4,24 +4,10 @@ The adapter calculates a Synthetix Index value in the currency selected
 
 ## Configuration
 
+- `DATA_PROVIDER_URL`: The adapter URL for the price data provider
```

Comment:
We'll probably want the `*_DATA_PROVIDER_URL` table in every composite that relies on Token Allocations

Reply:
I'll do the other composites that rely on Token Allocations in another PR. If this looks good let's get this one in so that you guys don't hit the errors when running `yarn setup`.